util/storage-samples/storage-owner-sample/src/main/java/org/fido/iot/storage/OwnerServiceInfoModule.java

```diff
@@ -30,25 +39,290 @@ public OwnerServiceInfoModule(DataSource dataSource) {
   @Override
   public List<ServiceInfoEntry> getServiceInfo(UUID uuid) {
     List<ServiceInfoEntry> serviceInfoEntries = new LinkedList<ServiceInfoEntry>();
+    String sviString = null;
 
-    String sql = "SELECT SVI_ID, MODULE_NAME, MESSAGE_NAME "
-        + "FROM GUID_OWNERSVI WHERE GUID = ? ORDER BY CREATED_AT ASC;";
+    // Add logic to identify devicetype
+
+    String deviceType = identifyDeviceType(uuid);
+
+    String sql = "SELECT OWNERSVI_STRING FROM DEVICE_TYPE_OWNERSVI_STRING WHERE DEVICE_TYPE = ?";
+    try (Connection conn = dataSource.getConnection();
+        PreparedStatement pstmt = conn.prepareStatement(sql)) {
+      pstmt.setString(1, deviceType);
+      try (ResultSet rs = pstmt.executeQuery()) {
+        if (rs.next()) {
+          sviString = rs.getString(1).trim();
+        }
+      }
+    } catch (SQLException e) {
+      throw new RuntimeException(e);
+    }
+
+    if (!sviString.equals("null")) {
+      sviString = insertModActivateSviEntry(sviString);
+      if (checkWgetInSviString(sviString) && isWgetVerificationEnabled()) {
+        sviString = insertWgetContentHash(sviString);
+      }
```

Comment:
Can we move this to the point when svi string was added? Since, this method gets called as many times as the number of serviceinfo message round-trips, this DB transaction is happening multiple times.

Reply:
Please remove the binary files in demo/<comp>/ that have been checked-in.

Reply:
That is already taken care of in this PR. The changes related to jar files is because I have deleted the previously tracked JAR files.

Reply:
Svi String insertion optimization done.